citi-onboarding/colisio
# Feature/home: Created the Home

# What I did

- Home created. Button without the link and the smooth effect.

>  Title, text, image and button lined up vertically in the middle of the home.

## How to test

1. Open the terminal and go to the folder of this project

2. In terminal:

  
   git checkout develop
   git pull
   git checkout <branch name>
   git pull origin <branch name>
   yarn install
   yarn start

3. In your browser go to http://localhost:port/

4. Check if the component looks just like the mockup in different screen sizes

5. Read the changed files section on GitHub

6. Test the things that you think are worth testing, even the ones that are not in this description
